pkg/scheduler/generic_scheduler_test.go

```diff
@@ -1513,12 +1531,19 @@ func TestPreferNominatedNodeFilterCallCounts(t *testing.T) {
 				t.Fatal(err)
 			}
 			snapshot := internalcache.NewSnapshot(nil, nodes)
-			scheduler := NewGenericScheduler(
+			scheduler := newScheduler(
 				cache,
+				nil,
+				nil,
+				nil,
+				nil,
+				nil,
+				nil,
+				nil,
 				snapshot,
-				schedulerapi.DefaultPercentageOfNodesToScore).(*genericScheduler)
+				schedulerapi.DefaultPercentageOfNodesToScore)
 
-			_, _, err = scheduler.findNodesThatFitPod(context.Background(), nil, fwk, framework.NewCycleState(), test.pod)
+			_, _, err = scheduler.findNodesThatFitPod(context.Background(), fwk, framework.NewCycleState(), test.pod)
 
```

Comment:
remove this blank line.

Reply:
I didn't add this blank lines, and It seems reasonable to isolate the "main test line" with two blank lines.

Reply:
Yes, this is not introduced by this pr. But I think refactoring is always worthy. And I still think this is a convention in go as adding an error judgement after  a function which has error returns.

Reply:
That makes sense.

removed four blank lines in `generic_scheduler_test.go`